src/app/pages/marketplace/index.js

```diff
@@ -2,54 +2,64 @@ import apiFetch from '@wordpress/api-fetch';
 import { useState, useEffect } from '@wordpress/element';
 import { useLocation } from 'react-router-dom';
 import classnames from 'classnames';
-import { Page } from "../../components/page";
-import { SectionContainer, SectionHeader, SectionContent } from "../../components/section";
-import { NewfoldRuntime } from "@newfold-labs/wp-module-runtime";
+import { Page } from 'App/components/page';
+import {
+	SectionContainer,
+	SectionHeader,
+	SectionContent,
+} from 'App/components/section';
+import { NewfoldRuntime } from '@newfold-labs/wp-module-runtime';
 // component sourced from marketplace module
 import { default as NewfoldMarketplace } from '../../../../vendor/newfold-labs/wp-module-marketplace/components/marketplace/';
 
 const MarketplacePage = () => {
-
-    // constants to pass to module
-    const moduleConstants = {
-		'supportsCTB': true,
-		'text': {
-			'title': __('Marketplace', 'bluehost-wordpress-plugin'),
-			'subTitle': __('Explore our featured collection of tools and services.', 'bluehost-wordpress-plugin'),
-			'error': __('Oops, there was an error loading the marketplace, please try again later.', 'bluehost-wordpress-plugin'),
-			'noProducts': __('Sorry, no marketplace items. Please, try again later.', 'bluehost-wordpress-plugin'),
-			'loadMore': __('Load More', 'bluehost-wordpress-plugin'),
-		}
+	// constants to pass to module
+	const moduleConstants = {
+		supportsCTB: true,
+		text: {
+			title: __( 'Marketplace', 'bluehost-wordpress-plugin' ),
+			subTitle: __(
+				'Explore our featured collection of tools and services.',
+				'bluehost-wordpress-plugin'
+			),
+			error: __(
+				'Oops, there was an error loading the marketplace, please try again later.',
+				'bluehost-wordpress-plugin'
+			),
+			noProducts: __(
+				'Sorry, no marketplace items. Please, try again later.',
+				'bluehost-wordpress-plugin'
```

Comment:
Can we make the "Please try again later" consistent?

I'd suggest `, please try again later.` for both.